docs/advanced.rst

```diff
@@ -946,6 +946,18 @@ in :file:`example-custom-exceptions.cpp``.
 This example also demonstrates how to create custom exception types
 with ``py::exception``.
 
+For declaring a simple, single exception translation that translates a C++
```

Comment:
I would probably move this whole text block to the very top of the section unchanged, otherwise it's a bit lost (i.e. "here's the simplest way to do it; for more complex handling, keep reading").